cms/toolbar/toolbar.py

```diff
@@ -24,7 +23,7 @@
 
 
 class CMSToolbarLoginForm(AuthenticationForm):
-    username = forms.CharField(label=_("Username"), max_length=100)
+    username = forms.CharField(max_length=100)
```

Comment:
Well yes, but the `username` in `AuthenticationForm` has a `max_length` of 254. I guess that should probably match whatever django sets. I wasn't sure if there was a reason it was set to 100 for the cms.